tests/Region/Inverted/DarkWorld/DeathMountain/East.php

```diff
@@ -0,0 +1,75 @@
+                        && $items->canBombThings()
+                        && $this->world->getRegion('East Death Mountain')->canEnter($locations, $items)) || ($this->world->config('canBootsClip', false)
+                        && $items->has('PegasusBoots')) ||
+                    $this->world->config('canOneFrameClipOW', false));
+        });
+
+        $this->locations["Hookshot Cave - Top Left"]->setRequirements(function ($locations, $items) {
+            return $items->has('Hookshot')
+                && ($items->canLiftRocks()
+                    || ($items->has('MagicMirror')
+                        && $items->canBombThings()
+                        && $this->world->getRegion('East Death Mountain')->canEnter($locations, $items)) || ($this->world->config('canBootsClip', false)
+                        && $items->has('PegasusBoots')) ||
+                    $this->world->config('canOneFrameClipOW', false));
+        });
+
+        $this->locations["Hookshot Cave - Bottom Left"]->setRequirements(function ($locations, $items) {
+            return $items->has('Hookshot')
+                && ($items->canLiftRocks()
+                    || ($items->has('MagicMirror')
+                        && $items->canBombThings()
+                        && $this->world->getRegion('East Death Mountain')->canEnter($locations, $items)) || ($this->world->config('canBootsClip', false)
+                        && $items->has('PegasusBoots')) ||
+                    $this->world->config('canOneFrameClipOW', false));
+        });
+
+        $this->locations["Hookshot Cave - Bottom Right"]->setRequirements(function ($locations, $items) {
+            return ($items->has('Hookshot')
+                || $items->has('PegasusBoots')) && ($items->canLiftRocks()
+                || ($items->has('MagicMirror')
+                    && $items->canBombThings()
+                    && $this->world->getRegion('East Death Mountain')->canEnter($locations, $items)) || ($this->world->config('canBootsClip', false)
+                    && $items->has('PegasusBoots')) ||
+                $this->world->config('canOneFrameClipOW', false));
+        });
+
+        $this->can_enter = function ($locations, $items) {
+            return ($this->world->getRegion('West Dark World Death Mountain')->canEnter($locations, $items)
+                && (!$this->world->config('region.cantTakeDamage', false)
+                    || $items->has('CaneOfByrna')
+                    || $items->has('Cape')
+                    || ($this->world->config('canBootsClip', false)
+                        && $items->has('PegasusBoots')) ||
+                    $this->world->config('canOneFrameClipOW', false))) || ($items->has('MagicMirror')
+                && $items->has('MoonPearl')
+                && $items->has('Hookshot')
+                && $this->world->getRegion('West Death Mountain')->canEnter($locations, $items));
+        };
+
+        return $this;
+    }
+}
```

Comment:
This file seems out of place, along with a bunch of other ones

Reply:
I don't even know when I would have put those there.
Removed them again.